certs/3072/include.am

```diff
@@ -6,4 +6,5 @@ EXTRA_DIST += \
          certs/3072/client-cert.der \
          certs/3072/client-cert.pem \
 		 certs/3072/client-key.der \
-         certs/3072/client-key.pem
+         certs/3072/client-key.pem \
+		 certs/3072/client-keyPub.der
```

Comment:
not going to hold this up based on white space but lets fix these hard tabs after the other PR's are passing again.

Reply:
Merge as-is. Looks good, Thanks David.